src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -875,14 +875,14 @@ public final void testExecuteLogOutput() throws Exception {
         final String auditFinishedMessage = bundle.getString(DefaultLogger.AUDIT_FINISHED_MESSAGE);
 
         final List<MessageLevelPair> expectedList = Arrays.asList(
-                new MessageLevelPair("checkstyle version .*", Project.MSG_VERBOSE),
+                new MessageLevelPair("Checkstyle version: .*", Project.MSG_VERBOSE),
+                new MessageLevelPair("Using config file:.*", Project.MSG_VERBOSE),
```

Comment:
It also contributes to the narrative in the logging info, since the config file is somewhat related to the version; both are meta-information and therefore logically belong together.

Reply:
follow up